datalad_dataset_addition_procedure.md

```diff
@@ -90,10 +95,16 @@ Previous entries in the ```.gitmodules``` file can be used as a guide.
 
 Choice of how to populate a new dataset will vary based on the special remote providing access to the data. The following procedure covers working with the [web](https://git-annex.branchable.com/tips/using_the_web_as_a_special_remote/) special remote. Alternative, more experimental options using other special remotes are documented [here] (https://github.com/CONP-PCNO/conp-documentation/datalad_dataset_addition_experimental.md)/.
 
-
 All commands presented in the following sections should be run from ```projects/<newprojectname>``` unless specified otherwise.
 
-Every dataset should include a valid DATS.json file compatible with the [CONP DATS format](https://github.com/CONP-PCNO/conp-documentation/CONP_DATS_field.md).
+### Requirements
+
+All datasets must include a `README.md` in the root directory.
+Adding meta-data about your dataset is required. 
```

Comment:
```suggestion
Adding metadata about your dataset is required. 
```

Note: next sentence, metadata is written with no `-`so to this suggestion is to stay consistent with the rest of the documentation and website.